samcli/cli/row_modifiers.py

```diff
@@ -0,0 +1,56 @@
+"""
+Row Definition and Row Modifiers to be used with `click.HelpFormatter write_dl methods`.
+"""
+
+from typing import NamedTuple, List
+
+from click import style
+
+
+class RowModifier:
```

Comment:
For completeness, should `RowModifier` be defined as an abstract class with an `apply` method?

Reply:
I thought about it, went with the current implementation to provide flexibility while writing.